measurement/dns/answer.go

```diff
@@ -27,7 +27,7 @@ import (
 // First two records from the response decoded by the probe, if they are
 // TXT or SOA; other RR can be decoded from "abuf"
 type Answer struct {
-    Data struct {
+    data struct {
```

Comment:
As discussed offline... But export `Data` remote the struct in a struct and delete 85% of this file, because you don't need those accessor methods.